### PR TITLE
Update a python-binding test script

### PR DESCRIPTION
**Pull request by nicelhc13**
Replace algorithms of SSSP and BFS with asynchronous on road network.
This commit also changes pagerank parameters.

### CODE REVIEW

File: scripts/bench_python_cpp_algos.py [hunk 1]
```diff
@@ -83,6 +91,7 @@ def run_sssp(property_graph: PropertyGraph, input_args, source_node_file):
 
         for source in sources:
             with time_block(f"sssp on {source}"):
+                source = source.strip("\n")
```
Comment (arthurp) [inline scripts/bench_python_cpp_algos.py]: I'm not sure what's going on here. source shouldn't be a string here.

**Review by arthurp — CHANGES_REQUESTED** [review]
One weird issue that could cause problems, so I want to fully understand it.

File: scripts/bench_python_cpp_algos.py [hunk 2]
```diff
@@ -128,7 +137,7 @@ def run_jaccard(property_graph: PropertyGraph, input_args):
 def run_pagerank(property_graph: PropertyGraph, _input_args):
     property_name = "NewProp"
 
-    tolerance = 0.000001
```
Comment (arthurp) [inline scripts/bench_python_cpp_algos.py]: This will conflict with #288.

File: scripts/bench_python_cpp_algos.py (hunk 1 above)
Comment (nicelhc13) [reply scripts/bench_python_cpp_algos.py]: Ah, yes. I have separate version of this script and it was string. My bad! Thank you for pointing this out!

**Review by roshandathathri — APPROVED** [review]
LGTM

**Review by arthurp — APPROVED** [review]

**Pull request merged**